warehouse/query-core/src/main/java/datawave/query/tables/edge/EdgeQueryLogic.java

```diff
@@ -693,6 +693,9 @@ public Set<String> getOptionalQueryParameters() {
         optionalParams.add(QueryParameters.DATATYPE_FILTER_SET);
         optionalParams.add(EdgeQueryConfiguration.INCLUDE_STATS);
         optionalParams.add(EdgeQueryConfiguration.DATE_RANGE_TYPE);
+        optionalParams.add(datawave.webservice.query.QueryParameters.QUERY_PAGETIMEOUT);
+        optionalParams.add(datawave.webservice.query.QueryParameters.QUERY_EXPIRATION);
+        optionalParams.add(datawave.webservice.query.QueryParameters.QUERY_MAX_RESULTS_OVERRIDE);
```

Comment:
Why is `QUERY_PAGESIZE` added as an optional param for the shard query logic but not the edge query logic?